publish_landing_page.sh

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+#!/bin/bash
+# Publishes the landing page to walrus sites.
+echo "Building site builder..." && \
+cargo build --release && \
+echo "Creating temporary landing page directory..." && \
+mkdir temp-landing-page && \
+cp -r portal/common/static/* temp-landing-page && \
+rm temp-landing-page/{\
+    index.html,\
+    404-page.template.html,\
+    sw.js,walrus-sites-portal-register-sw.js\
+    } && \
```

Comment:
We don't really need those files for the server portal, it's code for the service worker, and the 404 template is not used in the landing page directly either. 

I had to remove them because I copied the whole common/static directory which is used by all anything inside the portal dir.

Reply:
Created https://github.com/MystenLabs/walrus-sites/issues/179 just for this.